mkosi/__init__.py

```diff
@@ -2525,10 +2526,10 @@ def run_verb(args: MkosiArgs, presets: Sequence[MkosiConfig]) -> None:
         return bump_image_version()
 
     if args.verb == Verb.summary:
-        text = ""
-
-        for config in presets:
-            text += f"{summary(args, config)}\n"
+        if args.json:
+            text = json.dumps([p.to_dict() for p in presets], cls=MkosiJsonEncoder, indent=4, sort_keys=True)
```

Comment:
```suggestion
            text = json.dumps({"Presets": [p.to_dict() for p in presets]}, cls=MkosiJsonEncoder, indent=4, sort_keys=True)
```
I think I'd prefer this, because then we can extend the summary with other stuff later, e.g. when we want to add back the `MkosiArgs` or add version information or something like that.